rust/regorule-library/src/userinfo/v1.rego

```diff
@@ -0,0 +1,23 @@
+package stackable.opa.userinfo.v1
+
+# Lookup by (human-readable) username
+userInfoByUsername(username) := http.send({
```

Comment:
This doesn't actually import correctly, apparently OPA no longer allows this syntax? It was copied straight from the docs.

Arguably this would be an argument for at least running `opa check` on this in CI, but that isn't done at the moment.

Reply:
Nevermind, this was due asciidoc annotations that I had missed removing. Fixed in 1c74f89fa6694eb3903e61d1cc330dcd89fed9a9.